includes/express-checkout/class-wc-payments-express-checkout-button-handler.php

```diff
@@ -95,6 +95,7 @@ public function init() {
 		add_action( 'before_woocommerce_pay_form', [ $this, 'display_pay_for_order_page_html' ], 1 );
 		add_filter( 'woocommerce_gateway_title', [ $this, 'filter_gateway_title' ], 10, 2 );
 		add_action( 'woocommerce_checkout_order_processed', [ $this->express_checkout_helper, 'add_order_payment_method_title' ], 10, 2 );
+		add_filter( 'woocommerce_get_country_locale', [ $this->express_checkout_ajax_handler, 'modify_country_locale_for_express_checkout' ], 20 );
```

Comment:
What do you think about moving this filter registration to `includes/express-checkout/class-wc-payments-express-checkout-ajax-handler.php`, in its `init` method? This way the `modify_country_locale_for_express_checkout` method` can stay colocated with the filter registration.

Reply:
I agree, the thing is I tried once and the filter was not being triggered. But probably I was doing something wrong. I tried again and it worked fine so I refactored it in this commit [7937c7c](https://github.com/Automattic/woocommerce-payments/pull/10770/commits/7937c7c76f49505a74785ce34526431ac8220e8c)